### PR TITLE
separate vertical diffusion and surface flux

### PR DESCRIPTION
**Pull request by szy21**
This PR adds an option `nothing` to surface_scheme, so we can run simulations with vertical diffusion only.

### CODE REVIEW

**Comment by szy21** [comment]
bors try

File: examples/hybrid/sphere/baroclinic_wave_utilities.jl [hunk 1]
```diff
@@ -484,9 +486,13 @@ function vertical_diffusion_boundary_layer_tendency!(Yₜ, Y, p, t)
 
     if :ρe_tot in propertynames(Y.c)
         if !coupled
-            @. dif_flux_energy = Geometry.WVector(
-                surface_conditions.shf + surface_conditions.lhf,
-            )
+            if isnothing(surface_scheme)
```
Comment (charleskawczynski) [inline examples/hybrid/sphere/baroclinic_wave_utilities.jl:489]:
```suggestion
            if isnothing(p.surface_scheme)
```

File: examples/hybrid/sphere/baroclinic_wave_utilities.jl [hunk 2]
```diff
@@ -498,7 +504,11 @@ function vertical_diffusion_boundary_layer_tendency!(Yₜ, Y, p, t)
 
     if :ρq_tot in propertynames(Y.c)
         if !coupled
-            @. dif_flux_ρq_tot = Geometry.WVector(surface_conditions.E)
+            if isnothing(surface_scheme)
```
Comment (charleskawczynski) [inline examples/hybrid/sphere/baroclinic_wave_utilities.jl:507]:
```suggestion
            if isnothing(p.surface_scheme)
```

File: examples/hybrid/sphere/baroclinic_wave_utilities.jl (hunk 1 above)
Comment (szy21) [reply examples/hybrid/sphere/baroclinic_wave_utilities.jl:489]: 🤦‍♀️ Thanks!

**Review by charleskawczynski — APPROVED** [review]
I'm not sure off the top of my head if `@. dif_flux_energy *= 0` is supported, but these code changes seem overall fine to me

**Comment by szy21** [comment]
> I'm not sure off the top of my head if `@. dif_flux_energy *= 0` is supported, but these code changes seem overall fine to me

Would `@. dif_flux_energy = Ref(Geometry.WVector(FT(0)))` be better?

**Comment by szy21** [comment]
I will go ahead and merge it. I added a longrun which will tell us whether it works tomorrow:)

**Pull request merged**